debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouterConfigDefinition.java

```diff
@@ -0,0 +1,176 @@
+        }
+
+        @Override
+        public String getValue() {
+            return value;
+        }
+
+        /**
+         * Determine if the supplied value is one of the predefined options.
+         *
+         * @param value the configuration property value; may not be null
+         * @return the matching option, or null if no match is found
+         */
+        public static InvalidOperationBehavior parse(String value) {
+            if (value == null) return null;
+            value = value.trim();
+            for (InvalidOperationBehavior option : InvalidOperationBehavior.values()) {
+                if (option.getValue().equalsIgnoreCase(value)) return option;
+            }
+            return null;
+        }
+
+        /**
+         * Determine if the supplied value is one of the predefined options.
+         *
+         * @param value        the configuration property value; may not be null
+         * @param defaultValue the default value; may be null
+         * @return the matching option, or null if no match is found and the non-null default is invalid
+         */
+        public static InvalidOperationBehavior parse(String value, String defaultValue) {
+            InvalidOperationBehavior mode = parse(value);
+            if (mode == null && defaultValue != null) mode = parse(defaultValue);
+            return mode;
+        }
+    }
+
+    static final Field FIELD_EVENT_ID = Field.create("table.field.event.id")
+            .withDisplayName("Event ID Field")
+            .withType(ConfigDef.Type.STRING)
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDefault("id")
+            .withDescription("The field which contains the Event ID within the outbox table");
+
+    static final Field FIELD_EVENT_KEY = Field.create("table.field.event.key")
+            .withDisplayName("Event Key Field")
+            .withType(ConfigDef.Type.STRING)
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDescription("The field which contains the Event Key within the outbox table");
+
+    static final Field FIELD_EVENT_TYPE = Field.create("table.field.event.type")
+            .withDisplayName("Event Type Field")
+            .withType(ConfigDef.Type.STRING)
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDefault("type")
+            .withDescription("The field which contains the Event Type within the outbox table");
+
+    static final Field FIELD_PAYLOAD = Field.create("table.field.payload")
+            .withDisplayName("Event Payload Field")
+            .withType(ConfigDef.Type.STRING)
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDefault("payload")
+            .withDescription("The field which contains the Event Type within the outbox table");
```

Comment:
```suggestion
            .withDescription("The column which contains the payload within the outbox table");
```

Reply:
(should be "column" instead of "field" in all the option descriptions)

Reply:
Also the static properties?

Reply:
Depends on whether they refer to a column of a table or the field of a Kafka message (I'm not sure).